docs/workgroup/workgroup_charter.rst

```diff
@@ -107,27 +107,27 @@ tool.
 Budget
 ------
 
-The working group will not require a dedicated budget in its initial phase.
-If needed, the working group may ask the board of directors for any funds needed
+The Editorial Board will not require a dedicated budget in its initial phase.
+If needed, the Editorial Board may ask the board of directors for any funds needed
```

Comment:
To help disambiguate PSF WG vs things under the remit of the Council -- one of the original areas of confusion:

```suggestion
If needed, the Editorial Board may ask the PSF board of directors for any funds needed
```

A